### DISCUSSION TITLE
Input parameters for live-code module

### DISCUSSION THREAD

**Discussion by j4s0n-c** [noisedeck/noisedeck-docs#25] [Ideas]
Would be great to have some inputs to the live-code module. Maybe just a few sliders attached to a predefined vec4. Bonus if they can be named and range limited in code. Example of how they do it on the Erogenous Tones Structure eurorack module...
http://glsl.erogenous-tones.com/e#1341.0

**Comment by jesshewitt** [comment]
Great idea! This will be in the next release.
